### ISSUE TITLE
[1.19.2] Incompatibility with any mod including MixinExtras 0.2.0

### ISSUE THREAD

**Issue by TrkvonStpd1e11** [Thelnfamous1/Combat-Next#10]
Combat-Next requests a version dependency on MixinExtras that doesn't seem to include newer versions of the library,
which breaks it with any other mod that includes these versions.
```
[2012月2023 17:57:02.452] [main/ERROR] [net.minecraftforge.jarjar.selection.JarSelector/]: Failed to select jars for [ResolutionFailureInformation{failureReason=VERSION_RESOLUTION_FAILED, 
identifier=ContainedJarIdentifier[group=com.github.LlamaLad7, artifact=MixinExtras], 
sources=[
SourceWithRequestedVersionRange{source=[Mod File: D:\Program Files\PCL\.minecraft\versions\testing-1.19.2-Forge_43.3.5\mods\combat_next-1.19.2-1.0.1-all.jar], requestedVersionRange=[0.1.1,0.2), includedVersion=0.1.1}, 
SourceWithRequestedVersionRange{source=[Mod File: ], requestedVersionRange=[0.2.0,), includedVersion=0.2.0}
]}]
```

**Comment by RedPxnda** [comment]
Your issue may not be 1.20, but the fix would be the same: https://github.com/Thelnfamous1/Combat-Next/pull/12

**Comment by Thelnfamous1** [comment]
Fixed in 1.19.2-1.0.2.
